SQL/migrate/V016__Log_Tickets.sql

```diff
@@ -0,0 +1,17 @@
+--
+-- Adds a table for gathering statistics regarding tickets.
+--
+
+CREATE TABLE `ss13_tickets` (
+  `id` INT(11) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `game_id` VARCHAR(32) NOT NULL,
+  `message_count` INT(11) NOT NULL,
+  `admin_count` INT(11) NOT NULL,
```

Comment:
imho the following fields would be interesting / useful aswell:
 * admin_initial -> The initial admin that answered the ticket
 * admin_list -> A json list of all the admins in the ticket (Can then be searched with the json functions)
 * response_initial_at -> The time when the first response has been sent in the ticket